components/avatar/avatar.css

```diff
@@ -0,0 +1,39 @@
+@import url(/index.css);
+
+.avatar-img{
+    width: 70%;
+    height: 5rem;
+    border: 1px solid gray;
+    border-radius: 5px;
+    margin: auto;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 2rem;
+    background-color: aliceblue;
+}
+
+.avatar{
+    height: 3rem;
+    width:3rem;
+    border-radius: 50%;
+}
+
+.sm{
+    height: 2rem;
+    width:2rem;
+}
+
+.md{
+    height: 3rem;
+    width:3rem;
+}
+
+.lg{
+    height: 4rem;
+    width:4rem;
+}
+
+.sq{
+    border-radius: 0.3rem;
+}
```

Comment:
you are using 'rem' for border-radius, so this border-radius will change when you change the root font-size. Make sure if this is the behavior you want or set border-radius in pixels like you are doing in other places.

Reply:
done